### PR TITLE
Support ILR Econ Minor

### PR DESCRIPTION
**Pull request by Destaq**
### Summary 

This PR adds functionality for individuals to take the ILR Economics minor.

For details, please see [here](https://economics.cornell.edu/minor) and [here](https://cornell.app.box.com/s/bg41uhalv3j3pkkrhxgyg896mx2nm6yy).

### Test Plan 

image

You can go into your profile and add an ILR economics minor, then confirm that associated requirements are added in as expected.

## Notes

- Jest test case was updated to include this minor.
- This minor is only applicable to ILR students, but can be selected by any student. As such, an explicit note 'for ILR Students' was added to its description to ensure that non-ILR individuals do not pick it.

### CODE REVIEW

File: src/data/minors/econILR.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    name: 'Prerequisites',
+    description:
+      'MATH 1110 with grade C or better, ECON 1110 and ECON 1120 with grades B- or better.',
+    source: 'https://economics.cornell.edu/minor',
+    checker: includesWithSubRequirements(['MATH 1110'], ['ECON 1110'], ['ECON 1120']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1, 1],
+    slotNames: ['MATH 1110', 'ECON 1110', 'ECON 1120'],
+    checkerWarning:
+      'ECON 1110 or ECON 1120 will count toward the Economics Minor only if credit for the course appears on your Cornell transcript.',
```
Comment (plumshum) [inline src/data/minors/econILR.ts:20]: Aligned with the current rubric.

File: src/data/minors/econILR.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    source: 'https://economics.cornell.edu/minor',
+    checker: includesWithSubRequirements(['ECON 3030'], ['ECON 3040']),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1, 1],
+    slotNames: ['ECON 3030', 'ECON 3040'],
+  },
+  {
+    name: 'Statistics and Econometrics',
+    description:
+      'Option 1: ECON 3110 and ECON 3120. Option 2: ECON 3130 and ECON 3140. Must be taken at Cornell.',
```
Comment (plumshum) [inline src/data/minors/econILR.ts:34]: Looks good!

**Review by plumshum — APPROVED** [review]

File: src/data/index.ts [hunk 3]
```diff
@@ -529,6 +530,13 @@ const json: RequirementsJson = {
       advisors: eceMinorAdvisors,
       abbrev: 'ECE',
     },
+    ECONILR: {
+      name: 'Economics for ILR Students',
```
Comment (plumshum) [inline src/data/index.ts:534]: Another minor, information science in engineering, wrote the minoar like this
`Information Science [Engineering]`
should we keep the same semantics for consistency?

Comment (Destaq) [reply src/data/index.ts]: Updated, good catch.

**Pull request merged**